.github/workflows/flower-swift_sync.yml

```diff
@@ -0,0 +1,23 @@
+name: Sync flower-swift
+
+on:
+  push:
+    branches: ['main']
+    paths: ['src/swift/**']
+  pull_request:
+    branches: ['main']
+    paths: ['src/swift/**']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Pushes src/swift to flower-swift repository
+        uses: cpina/github-action-push-to-another-repository@main
```

Comment:
Do not use @main but instead a specific git commit, e.g. "cbe757f", which is release v1.7. We do not want to enable a dependency which might get compromised to get full access to all our environment variables.